includes/in-person-payments/class-wc-payments-in-person-payments-receipt-service.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Class WC_Payments_In_Person_Payment_Print_Receipt_Service
+ *
+ * @package WooCommerce\Payments
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class handling in person payments receipts.
+ */
+class WC_Payments_In_Person_Payments_Receipt_Service {
+
+	/**
+	 * Renders the receipt template.
+	 *
+	 * @param  array    $settings Merchant settings.
+	 * @param  WC_Order $order Order instance.
+	 * @param  array    $charge Charge data.
+	 *
+	 * @return void
+	 */
+	public static function render_receipt( array $settings, WC_Order $order, array $charge ) {
+
+		$order_data = [
+			'id'           => $order->get_id(),
+			'currency'     => $order->get_currency(),
+			'subtotal'     => $order->get_subtotal(),
+			'line_items'   => $order->get_items(),
+			'coupon_lines' => $order->get_items( 'coupon' ),
+			'tax_lines'    => $order->get_items( 'tax' ),
+			'total'        => $order->get_total(),
+		];
+
+		wc_get_template(
+			'html-in-person-payment-receipt.php',
+			[
+				'amount_captured'        => $charge['amount_captured'] / 100,
+				'coupon_lines'           => $order_data['coupon_lines'] ?? [],
+				'business_name'          => $settings['business_name'],
+				'line_items'             => self::format_line_items( $order_data ),
+				'order'                  => $order_data,
+				'payment_method_details' => $charge['payment_method_details']['card_present'],
+				'receipt'                => $charge['payment_method_details']['card_present']['receipt'],
+				'support_address'        => $settings['support_info']['address'],
+				'support_email'          => $settings['support_info']['email'],
+				'support_phone'          => $settings['support_info']['phone'],
+				'tax_lines'              => $order_data['tax_lines'] ?? [],
+			],
+			'',
+			__DIR__ . '/templates/'
+		);
+	}
+
+	/**
+	 * Format line items
+	 *
+	 * @param  array $order the order.
+	 * @return array
+	 */
+	private static function format_line_items( $order ) {
```

Comment:
We can declare parameter and return types right in the method signature.

Reply:
Done!